### PR TITLE
Removed Gravatar related code

### PR DESCRIPTION
**Pull request by diliger**
as described in issue #48

### CODE REVIEW

**Comment by warpech** [comment]
I think this is not all. Everything that relates to `ImageURL` property of [Somebody](https://github.com/StarcounterPrefabs/Simplified/blob/ff4e987341157f6575758cb05b063b451f9d5293/Ring1/Somebody.cs#L15) should be removed.

Keep in mind this does not affect Images app at all, because that app uses `Illustration` and `Content`. 

`ImageURL`is probably not used anywhere besides UserAdmin and should be marked as obsolete in Simplified.

**Comment by diliger** [comment]
Did that.

File: src/UserAdmin/Helpers/Utils.cs [hunk 1]
```diff
@@ -71,18 +71,5 @@ public class Utils {
                 return false;
             }
         }
-
-        static public string GetGravatarUrl(string email) {
-
-            using (MD5 md5Hash = MD5.Create()) {
-                byte[] data = md5Hash.ComputeHash(Encoding.UTF8.GetBytes(email.Trim().ToLowerInvariant()));
```
Comment (warpech) [inline src/UserAdmin/Helpers/Utils.cs]: Please clean up unused `Using` at the top of this file.

**Comment by diliger** [comment]
Done.

**Comment by warpech** [comment]
Great, thanks! Merging

**Pull request merged**